src/NetscriptFunctions/Bladeburner.ts

```diff
@@ -27,6 +27,21 @@ export function NetscriptBladeburner(): InternalAPI<INetscriptBladeburner> {
       throw helpers.errorMessage(ctx, "You must be a member of the Bladeburner division to use this API.");
     return bladeburner;
   };
+  const checkSleeveAPIAccess = function (ctx: NetscriptContext) {
```

Comment:
I'm pretty sure this function already exists elsewhere... you should import that version.

Reply:
@d0sboots 
Everything has been updated.  I was going to export the last 2 but they were defined within a function already.  Since you've given permission though, I've removed those 2 from their main function, defined them above it and exported them.